src/components/breakpoint.scss

```diff
@@ -1,4 +1,4 @@
 $screen-sm-min: 768px;
-$screen-md-min: 992px;
+$screen-md-min: 900px;
```

Comment:
Dont change queries because it suits your needs. Either choose bootstrap ones ->
![CleanShot 2024-03-04 at 17 14 16](https://github.com/GLObus303/road-to-star/assets/12488648/27688144-c7bf-4450-b47e-eb18ea6d105a)

or chrome ones and stick to them